plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/taxonomy-filter/inspector.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, SelectControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { getSetting } from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import type { EditProps, TaxonomyItem } from './types';
+import { DisplayStyleSwitcher } from '../../components/display-style-switcher';
+import metadata from './block.json';
+import { updateFilterHeading } from '../../utils/update-filter-heading';
+import { getTaxonomyLabel } from './utils';
+
+export const Inspector = ( {
```

Comment:
Personally, I find name `Inspector` confusing. If possible, I'd rather avoid naming `InspectorControls` as `Inspector`. I know it comes from conflict with `InspectorControls` from `'@wordpress/block-editor'`. But if you don't mind, what about changing this export to default export:

```suggestion
export default function ( {
```

and import it as `InspectorControls` or give it more explicit name:

```suggestion
export const TaxonomyFilterInspectorControls = ( {
```

Reply:
Will do :D